flink-clients/src/main/java/org/apache/flink/client/program/StandaloneClusterClient.java

```diff
@@ -70,6 +76,8 @@ public GetClusterStatusResponse getClusterStatus() {
 			} else {
 				throw new RuntimeException("Received the wrong reply " + result + " from cluster.");
 			}
+		} catch (FlinkRuntimeException | FlinkException e) {
```

Comment:
Hi! @tillrohrmann , We can get there even FlinkException,. I just tried use more common approach for processing exception while getting Cluster status and prevent unpredictable test failure.  Maybe it would better throw there only FlinkRuntimeException in all cases. It makes job fail without any timeout.  What do you think?